core/src/geometry/point.rs

```diff
@@ -234,6 +238,21 @@ impl Point {
     pub const fn component_div(self, other: Self) -> Self {
         Self::new(self.x / other.x, self.y / other.y)
     }
+
+    /// Returns a point with swapped x and y coordinates.
```

Comment:
Nit: we use uppercase in a couple of other places at least

```suggestion
    /// Returns a point with swapped X and Y coordinates.
```